examples/bitcoin_psbt/test/test.ts

```diff
@@ -1,45 +1,23 @@
+import * as dns from 'node:dns';
+dns.setDefaultResultOrder('ipv4first');
+
 import { getCanisterId } from 'azle/dfx';
+import { whileRunningBitcoinDaemon } from 'azle/examples/basic_bitcoin/test/bitcoin-daemon';
+import { getTests } from 'azle/examples/basic_bitcoin/test/tests';
 import { runTests } from 'azle/test';
-import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
-import { existsSync, rmSync } from 'fs-extra';
-
-import { getTests } from './tests';
-
-const canisterId = getCanisterId('basic_bitcoin');
-
-export async function whileRunningBitcoinDaemon(
-    callback: () => Promise<boolean> | void
-) {
-    const bitcoinDaemon = await startBitcoinDaemon();
-    await callback();
-    bitcoinDaemon.kill();
-}
 
-async function startBitcoinDaemon(): Promise<ChildProcessWithoutNullStreams> {
-    if (existsSync(`.bitcoin/regtest`)) {
-        rmSync('.bitcoin/regtest', { recursive: true, force: true });
-    }
-    const bitcoinDaemon = spawn('.bitcoin/bin/bitcoind', [
-        `-conf=${process.cwd()}/.bitcoin.conf`,
-        `-datadir=${process.cwd()}/.bitcoin`,
-        '--port=18444'
-    ]);
+export const P2WPKH_ADDRESS_FORM =
```

Comment:
Let's make a tests.ts file and put the constants in there and export the getTests from there

Reply:
done